ibis/pyspark/compiler.py

```diff
@@ -1040,12 +1013,12 @@ def compile_lead(t, expr, scope, *, window, **kwargs):
 
 @compiles(ops.MinRank)
 def compile_rank(t, expr, scope, *, window, **kwargs):
-    return F.rank().over(window).astype('long') - F.lit(1)
+    return F.rank().over(window).astype('long') - 1
```

Comment:
@icexelloss could you provide a docstring here using numpy format? https://numpydoc.readthedocs.io/en/latest/format.html